src/client/src/components/pages/participants/addParticipant/staticData.js

```diff
@@ -274,7 +284,14 @@ export const fields = [
 ];
 
 export const validationForm = fields => {
+  console.log(fields);
   for (const key in fields) {
     if (fields[key] === "") return `Please Check ${key}`;
   }
+
+  if (!validator.isEmail(fields['email'])) {
+    return `${fields['email']} Is Not A Valid Email.`
```

Comment:
I think it looks better to write your validation messages as a sentence. So, only the first word, as well as certain names, are capitalised.
```suggestion
    return `${fields['email']} is not a valid email.`
```